core/metrics-core-impl/src/test/java/org/hawkular/metrics/core/impl/DelegatingDataAccess.java

```diff
@@ -20,18 +20,15 @@
 import java.util.Map;
 import java.util.Set;
 
+import com.datastax.driver.core.ResultSet;
```

Comment:
Another thing, update your hawkular-build-tools and reapply the settings.jar to get correct imports. They're messed up in every committed file.